manifests/flannel.yaml

```diff
@@ -98,8 +98,7 @@ spec:
       nodeSelector:
         beta.kubernetes.io/arch: amd64
       tolerations:
-      - key: node-role.kubernetes.io/master
-        operator: Exists
+      - operator: Exists
```

Comment:
How does this work without a key? What does it do

Reply:
Looks good.